calkit/cli/overleaf.py

```diff
@@ -386,12 +393,15 @@ def sync(
             if diff:
                 typer.echo("Applying to project repo")
                 process = subprocess.run(
-                    ["git", "apply", "--directory", wdir, "-"],
+                    ["git", "apply", "--reject", "--directory", wdir, "-"],
                     input=diff,
                     text=True,
                 )
                 if process.returncode != 0:
-                    raise_error("Failed to apply diff")
+                    if force:
+                        typer.echo("Failed to apply Overleaf diff to project repo. Proceeding to push project changes to Overleaf anyway.")
```

Comment:
These lines being so long fail the formatting check. You can wrap/continue them like:

```suggestion
                        typer.echo(
                            "Failed to apply Overleaf diff to project repo. "
                            "Proceeding to push project changes to Overleaf anyway."
                        )
```